### PR TITLE
Export symbols

### PR DESCRIPTION
**Pull request by YoannPitarch**
Related to issue #131

### CODE REVIEW

**Comment by rusty1s** [comment]
Thanks for the PR. I am currently on vacation and will take a look next week. Let me know if there is any problem in fixing the CI errors.

**Review by rusty1s — APPROVED** [review]
@YoannPitarch Thank you very much for the PR. This looks good to me, although I am unsure why Windows tests are failing. Do you have an idea?

**Comment by YoannPitarch** [comment]
> @YoannPitarch Thank you very much for the PR. This looks good to me, although I am unsure why Windows tests are failing. Do you have an idea?

You're welcome. I do not get why Windows tests are failing as well. I've managed to compile the lib on both Windows and Linux. However, I must say that linking this libs (but it's also true for `pytorch-sparse` and `pytorch-cluster`) to torchlib on Windows works fine in Debug mode (compiling the lib in Debug mode and using the debug version of torchlib) while its fails in Release mode. 
It is not clear to me if this is related to the PR and if that can explain why  Windows tests are failing.

File: setup.py [hunk 1]
```diff
@@ -34,6 +34,10 @@ def get_extensions():
 
     for main, suffix in product(main_files, suffices):
         define_macros = []
+
+        if sys.platform == 'win32':
+            define_macros += [('torchsparse_EXPORTS', None)]
```
Comment (rusty1s) [inline setup.py:39]:
```suggestion
            define_macros += [('torchcluster_EXPORTS', None)]
```

**Comment by rusty1s** [comment]
I think I fixed it :)

**Comment by YoannPitarch** [comment]
Thanks !

**Pull request merged**